src/app-layer-ssl.h

```diff
@@ -206,6 +209,10 @@ typedef struct SSLStateConnp_ {
 
     uint16_t session_id_length;
 
+    uint8_t random[TLS_RANDOM_LEN];
+    /* flag to indicate that the random array is set */
+    bool random_set;
```

Comment:
this 1 byte bool field will create a 7 byte gap in the data structure due to alignment requirements of the next pointer field. It could be solved by either moving the bool to after the cert_log_flag field or by making this a set of flags for the `SSLState::flags` field

Reply:
Thank you. :) Haven't thought of alignments since Netfilter. 😅